src/Logic/calculator.js

```diff
@@ -0,0 +1,37 @@
+class Calc {
+  constructor() {
+    this.result = null;
+  }
+
+  add(arg1, arg2) {
+    this.result = arg1 + arg2;
+    return this.result;
+  }
+
+  plusOrMinus(arg1, arg2) {
+    this.result = [arg1 + arg2, arg1 - arg2];
+    return this.result;
+  }
+
+  subtract(arg1, arg2) {
+    this.result = arg1 - arg2;
+    return this.result;
+  }
+
+  multiply(arg1, arg2) {
+    this.result = arg1 * arg2;
+    return this.result;
+  }
+
+  divide(arg1, arg2) {
+    this.result = arg1 / arg2;
+    return this.result;
+  }
+
+  modulus(arg1, arg2) {
+    this.result = arg1 % arg2;
+    return this.result;
+  }
+}
+
+export default Calc;
```

Comment:
- [ ] I like the fact that you stretch yourself to write your own functions for a specific operation.
However, it will be nice if you could follow the project requirements. You were required to make a copy of this [calculate file](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js). You were supposed to copy the contents of that file paste it into your created file.
Please make this change. And don't forget to verify the calculation works as expected.